src/explorer.ts

```diff
@@ -400,6 +401,17 @@ class KubernetesSelectsPodsFolder extends KubernetesResourceFolder {
     }
 }
 
+class KubernetesCRDFolder extends KubernetesFolder {
+    constructor(readonly kind: kuberesources.ResourceKind) {
+        super(kind.abbreviation, kind.pluralDisplayName);
+    }
+
+    async getChildren(kubectl: Kubectl, host: Host): Promise<KubernetesObject[]> {
+        const objects = await kubectlUtils.getGlobalResources(kubectl, "crd");
+        return objects.map((obj) => new KubernetesResourceFolder(new kuberesources.ResourceKind(obj.metadata.name, obj.metadata.name, obj.metadata.name, obj.metadata.name)));
```

Comment:
I hadn't realised before, but the CRD resource type has `names.plural`, `names.singular` and `names.kind` (and an optional `names.shortNames` array).  I think we should _at minimum_ use `names.kind` as the manifest kind (argument 3), and consider using `names.singular` as the abbreviation (argument 4).  Arguments 1 and 2 aren't completely clear cut, because the singular and plural in the manifest are programmatic not display names, but they would arguably still be friendlier than the CRD name in all its dotted glory.  Thoughts?

Reply:
hm, the `obj` is of type `KubernetesResource` which only has `.metadata.name` and `.kind` properties. Should we create a `getCRDs` method that returns a `CRDInfo` array (or something like that)?